### DISCUSSION TITLE
建议添加语音朗读中英文功能

### DISCUSSION THREAD

**Discussion by yuan8848** [immersive-translate/immersive-translate#1169] [Ideas]
### 新功能 | New Feature

1.建议添加语音朗读中英文功能
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.翻译了 我想听下英文朗读,不想用其他插件 希望一起集成
2.
3.

**Comment by kun2001github** [comment]
我也这么认为，这样特别方便，就不需要再安装其他插件
